From eeba29fa7321bcec47f4d4ff0d6f32f31bd11e3d Mon Sep 17 00:00:00 2001
From: Dale Curtis <dalecurtis@chromium.org>
Date: Tue, 24 Jul 2012 18:11:43 -0700
Subject: [PATCH] Disable AAC sample skipping.

Currently non-Apple encoders aren't supported and this change chops
off more samples than necessary for these files.

See http://ffmpeg.org/pipermail/ffmpeg-devel/2012-July/127834.html

BUG=none
TEST=http://rtoy.github.com/webaudio/codec-tests/plot-audio.html
---
 libavformat/mov.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libavformat/mov.c b/libavformat/mov.c
index b4295f8..1795707 100644
--- a/libavformat/mov.c
+++ b/libavformat/mov.c
@@ -2979,7 +2979,7 @@ static int mov_read_header(AVFormatContext *s)
         AVStream *st = s->streams[i];
         MOVStreamContext *sc = st->priv_data;
         if(st->codec->codec_type == AVMEDIA_TYPE_AUDIO && st->codec->codec_id == CODEC_ID_AAC) {
-            sc->start_pad = 2112;
+            sc->start_pad = 0; // 2112; TODO(dalecurtis): Disabled until non-Apple encoders are supported.
             st->skip_samples = sc->start_pad;
         }
     }
-- 
1.7.11.2

